middleware/rewrite/README.md

```diff
@@ -23,6 +23,10 @@ rewritten; e.g., to rewrite CH queries to IN use `rewrite class CH IN`.
 When the FIELD is `name` the query name in the message is rewritten; this
 needs to be a full match of the name, e.g., `rewrite name miek.nl example.org`.
 
+When the FIELD is `edns0` an EDNS0_LOCAL option can be appended to the request;
+e.g., to force a code on all requests, `rewrite edns0 replace_or_append 0xffad 55555555555555555555555555555555`;
```

Comment:
Yeah. We couldn't think of a better name that captured the semantics. I guess we could change to DB-like terminology:

insert - append it
update - change it if it is there
upsert - made up word meaning update or insert if not found